Source/DiabloUI/credits.cpp

```diff
@@ -199,6 +200,10 @@ bool TextDialog()
 	CreditsRenderer creditsRenderer;
 	bool endMenu = false;
 
+	bool wasCursorVisible;
+	if (IsHardwareCursorEnabled())
+		wasCursorVisible = SetHardwareCursorVisible(false);

```

Comment:
How come we aren't able to use `ShouldShowCursor()` in situations like this?

Reply:
This is the main menu code, `ShouldShowCursor` is game-specific, though it could probably be improved in the future.

Reply:
Then I'm pretty sure this can simply be replaced with `!sgbControllerActive`

Reply:
Yeah I guess, let me change that

Reply:
Turned out to be even simpler than that and fixed some subtle bugs too. Done.